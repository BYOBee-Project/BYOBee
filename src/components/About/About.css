.about {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 80%;
}

.about-title {
  font-family: 'Londrina Solid', cursive;
  font-size: 6vh;
  margin: 8vh;
}

.about-bios {
  display: flex;
  gap: 2vw;
  justify-content: space-around;
  align-items: flex-start;
}

.about-bio-card {
  width: 20vw;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding: 1vh 2vw 1vh 2vw;
  border-radius: 10px;
  background-color: hsla(4, 82%, 67%, 0.827);
}

.about-bio-card-title {
  font-family: 'Londrina Solid', cursive;
  font-size: 3vh;
}

.about-bio-card-image {
  width: 10vw;
  border-radius: 5px;
}

.credit {
  margin: 3vh;
}
