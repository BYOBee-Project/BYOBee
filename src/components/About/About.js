import React from 'react';
import './About.css';
import emma from '../../assets/emma.jpeg';
import bumblebee from '../../assets/bumblebee.jpeg';
import michelle from '../../assets/michelle.jpg';
import zach from '../../assets/zachphoto.jpg';

export default function About() {
  return (
    <div>
      <h1>About The Hive</h1>
      <div className="bios">
        <div className="bio-card">
          <h3>Adria Ivanitsky</h3>
          <img src={bumblebee} />
          <p>
            Adria is a web developer in portland, OR. she loves to playing soccer, making music, and
            being annoying in the zoom chat. also bees
          </p>
        </div>
        <div className="bio-card">
          <h3>Emma Egstad</h3>
          <img src={emma} />
          <p>
            Emma is a web developer in Portland, OR. She loves each and every bee, but especially
            super fuzzy digger bees. In addition to bees, Emma loves CSS, array methods, and weekly
            builds (just kidding).
          </p>
        </div>
        <div className="bio-card">
          <h3>Michelle Nygren</h3>
          <img src={michelle} />
          <p>
            Michelle is a web developer in Portland, OR. She loves making things from beeswax
            (thanks, honeybees!), and using honey in her baking. In addition to bees, Michelle loves
            keeping Julie on her toes with her hand raised in class.
          </p>
        </div>
        <div className="bio-card">
          <h3>Zachary Caldwell</h3>
          <img src={zach} />
          <p>
            Zach is a web developer in Portland, OR. He loves foraging for mushrooms with his dog.
            Zach loves when he forgets he is not muted and burps loudly over zoom. He loves when
            projects dont have CSS in the grading rubric.
          </p>
        </div>
      </div>
    </div>
  );
}
