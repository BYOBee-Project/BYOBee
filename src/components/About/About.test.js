import { render, screen } from '@testing-library/react';
import About from './About.js';

test('About page should render about us stuff', async () => {
  const container = render(<About />);
  screen.findByText('About the hive');
  expect(container).toMatchSnapshot();
});
