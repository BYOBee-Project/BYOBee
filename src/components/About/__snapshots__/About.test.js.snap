// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page should render about us stuff 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="about"
      >
        <h1
          class="about-title"
        >
          About The Hive
        </h1>
        <div
          class="about-bios"
        >
          <div
            class="about-bio-card"
          >
            <h3
              class="about-bio-card-title"
            >
              Adria Ivanitsky
            </h3>
            <img
              class="about-bio-card-image"
              src="linkedin.jpeg"
            />
            <p>
              Adria is a web developer in Portland, OR. She loves playing soccer, making music, and being annoying in the Zoom chat. Also... bees.
            </p>
          </div>
          <div
            class="about-bio-card"
          >
            <h3
              class="about-bio-card-title"
            >
              Emma Egstad
            </h3>
            <img
              class="about-bio-card-image"
              src="emma.jpeg"
            />
            <p>
              Emma is a web developer in Portland, OR. She loves each and every bee, but especially super fuzzy digger bees. In addition to bees, Emma loves CSS, array methods, and weekly builds (just kidding).
            </p>
          </div>
          <div
            class="about-bio-card"
          >
            <h3
              class="about-bio-card-title"
            >
              Michelle Nygren
            </h3>
            <img
              class="about-bio-card-image"
              src="michelle.jpg"
            />
            <p>
              Michelle is a web developer in Portland, OR. She loves making things from beeswax (thanks, honeybees!), and using honey in her baking. In addition to bees, Michelle loves keeping Julie on her toes with her hand raised in class.
            </p>
          </div>
          <div
            class="about-bio-card"
          >
            <h3
              class="about-bio-card-title"
            >
              Zachary Caldwell
            </h3>
            <img
              class="about-bio-card-image"
              src="zachphoto.jpg"
            />
            <p>
              Zach is a web developer in Portland, OR. He loves foraging for mushrooms with his dog. Zach loves when he forgets he is not muted and burps loudly over zoom. He loves when projects dont have CSS in the grading rubric.
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="about"
    >
      <h1
        class="about-title"
      >
        About The Hive
      </h1>
      <div
        class="about-bios"
      >
        <div
          class="about-bio-card"
        >
          <h3
            class="about-bio-card-title"
          >
            Adria Ivanitsky
          </h3>
          <img
            class="about-bio-card-image"
            src="linkedin.jpeg"
          />
          <p>
            Adria is a web developer in Portland, OR. She loves playing soccer, making music, and being annoying in the Zoom chat. Also... bees.
          </p>
        </div>
        <div
          class="about-bio-card"
        >
          <h3
            class="about-bio-card-title"
          >
            Emma Egstad
          </h3>
          <img
            class="about-bio-card-image"
            src="emma.jpeg"
          />
          <p>
            Emma is a web developer in Portland, OR. She loves each and every bee, but especially super fuzzy digger bees. In addition to bees, Emma loves CSS, array methods, and weekly builds (just kidding).
          </p>
        </div>
        <div
          class="about-bio-card"
        >
          <h3
            class="about-bio-card-title"
          >
            Michelle Nygren
          </h3>
          <img
            class="about-bio-card-image"
            src="michelle.jpg"
          />
          <p>
            Michelle is a web developer in Portland, OR. She loves making things from beeswax (thanks, honeybees!), and using honey in her baking. In addition to bees, Michelle loves keeping Julie on her toes with her hand raised in class.
          </p>
        </div>
        <div
          class="about-bio-card"
        >
          <h3
            class="about-bio-card-title"
          >
            Zachary Caldwell
          </h3>
          <img
            class="about-bio-card-image"
            src="zachphoto.jpg"
          />
          <p>
            Zach is a web developer in Portland, OR. He loves foraging for mushrooms with his dog. Zach loves when he forgets he is not muted and burps loudly over zoom. He loves when projects dont have CSS in the grading rubric.
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
